shell/scripts/extension-test/test-plugins-parse-login

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const readline = require('readline');
+
+const rl = readline.createInterface({
```

Comment:
Could you please add a comment about which stream we are interacting as there's no login command around?